### ISSUE TITLE
Turbine Interaction Improvments

### ISSUE THREAD

**Issue by eslickj** [CCSI-Toolset/FOQUS#769]
These are some things that would make the Turbine remote/cloud computing component more useful and usable.  If we start working on these we can break them out into new issues as needed.

1.) Run optimization and UQ completely on the cloud.  That way you can easily start up multiple optimzation or UQ runs in parallel and don't need to keep your machine running.   This should be fairly easy to implement.  Instead of just single flowsheet runs, we can allow optimization and UQ model.  You can download the FOQUS session when it's done for the results.  We can probably save the session periodically while it's running so you can check progress. 

2.) We currently have a local and remote mode.  Local isn't really any different than remote, and both work.  We should switch to remote only. This will make things easier to maintain, and make it more likely the users will try out the cloud stuff because the workflow is no different.  The local mode can be decoupled from the main foqus program and moved into the consumer.  This shouldn't be to hard, since everything already works.  The main issue is that FOQUS and the FOQUS consumer are currently the same program.  If we do this, we'd probably want to split them in two.  The consumer would run flowsheets, UQ and optimization, but not have any GUI elements.  FOQUS would then be the UI and talk to the consumer.  The only difficulty is splitting the program, and it may be a lot of work.  Just from a user perspective, we can achieve the same function, just by removing the local option.  We could then take our time to sort it out in the background.

3.) If users are using cloud computing they may expect Excel to be available in addition to Aspen.  In rare cases Excel may even be the slow step, so each worker having excel would be handy.  Not sure what we can do about this, but it a blocking issue for a lot of people.

**Comment by boverhof** [comment]
Moved issue 1 to new issue #800

**Comment by ksbeattie** [comment]
Replaced by #800
